api/tests/unit/domain/usecases/sessions-mass-import/validate-sessions_test.js

```diff
@@ -41,9 +41,10 @@ describe('Unit | UseCase | sessions-mass-import | validate-sessions', function (
 
   context('when sessions are valid', function () {
     context('when user has certification center membership', function () {
-      it('should validate every session one by one', async function () {
+      it('should validate every session one by one and return cachedValidatedSessionsKey, sessionsCount, emptySessionsCount and candidatesCount', async function () {
```

Comment:
```suggestion
      it('should validate every session one by one and return a sessionsMassImportReport', async function () {
```

J'ai l'impression que la notion de _report_ commence à arriver. Plutôt que de renvoyer un POJO, on pourraît commencer à renvoyer un model, WDYT ?

Reply:
J'approuve !